tests/storm.thrift

```diff
@@ -1,4 +1,4 @@
-#!/usr/local/bin/thrift --gen java:beans,nocamel,hashcode
+// #!/usr/local/bin/thrift --gen java:beans,nocamel,hashcode
```

Comment:
I noticed this earlier. technically thrift does not support hash comments, only C-style (`//` and `/* */`), so it's good the new parser caught it.